<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Process Scheduler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="processTable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>1001</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QFrame" name="controlFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>1001</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buttonRandomProcess">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Random process</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMsQuant">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ms / Tick&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineProcessPlan">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>621</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferUppercase|Qt::ImhUppercaseOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelProcessPlan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Process plan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddProcess">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMsQuant">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
    <widget class="QComboBox" name="cbChooseAlgo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>FCFS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RR</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelAlgo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Algorithm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonTick">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tick</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonClear">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelViewTime">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;00:00:00.000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelViewCountTicks">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCountTIcks">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Count ticks: </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelViewPID">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>70</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurrentExPID">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>71</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current executed PID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSystemTime">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;System Time: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTicks">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelTicksPerProc">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ticks per process</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
